android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java

```diff
@@ -187,6 +189,9 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
                 (BraveDialogPreference) findPreference(PREF_FINGERPRINTING_PROTECTION);
         mFingerprintingProtectionPref.setOnPreferenceChangeListener(this);
 
+        mFingerprntLanguagePref = (ChromeSwitchPreference) findPreference(PREF_FINGERPRINT_LANGUAGE);
```

Comment:
@sujitacharya2005 how are you syncing the pref value here ? So after changing the value, how you are checking the value and setting the pref value here ?

Reply:
@deeppandya 
1. From SwitchPreference when change value onPreferenceChange will call , there updating UserPrefs.
2. in feature.cc it's set initial value disabled
const base::Feature kBraveReduceLanguage{"BraveReduceLanguage",
                                         base::FEATURE_DISABLED_BY_DEFAULT};
3. From UI side also initial value SwitchPreference is false
<org.chromium.components.browser_ui.settings.ChromeSwitchPreference
            android:defaultValue="false" />

4. Now I need to verify that UserPrefs setting in native code happening properly
UserPrefs.get(Profile.getLastUsedRegularProfile())
                    .setBoolean(BravePref.REDUCE_LANGUAGE_ENABLED, (boolean) newValue);
5. My understanding like once setting this UserPrefs value automatically native will take care the feature
so I am not doing any syncing logic.
Please let me know if anything I am missing

Reply:
@deeppandya 
Updated initial UI SwitchPreference state based on native value.